src/test/java/org/opensearch/security/SlowIntegrationTests.java

```diff
@@ -48,10 +50,15 @@
 import org.opensearch.security.test.helper.cluster.ClusterConfiguration;
 import org.opensearch.security.test.helper.file.FileHelper;
 import org.opensearch.security.test.helper.rest.RestHelper;
+import org.opensearch.security.util.Repeat;
+import org.opensearch.security.util.RepeatRule;
 import org.opensearch.transport.Netty4ModulePlugin;
 
 public class SlowIntegrationTests extends SingleClusterTest {
 
+    @Rule
```

Comment:
This rule was pulled from [here](https://stackoverflow.com/a/28345802).

I could not use JUnit5's [@RepeatedTest](https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/RepeatedTest.html) annotation.